pr_check.sh

```diff
@@ -1,107 +1,21 @@
+#!/bin/bash
 
-export LANG=en_US.utf-8
-export LC_ALL=en_US.utf-8
+APP_NAME="rbac"  # name of app-sre "application" folder this component lives in
+COMPONENT_NAME="rbac"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
+IMAGE="quay.io/cloudservices/rbac"  # the image location on quay
 
-#
-# Build PR image
-#
+IQE_PLUGINS="rbac"  # name of the IQE plugin for this APP
+IQE_MARKER_EXPRESSION="rbac_smoke"  # This is the value passed to pytest -m
+IQE_FILTER_EXPRESSION=""  # This is the value passed to pytest -k
 
-IMAGE="quay.io/cloudservices/rbac"
-IMAGE_TAG=$(git rev-parse --short=7 HEAD)
-DOCKER_CONF="$PWD/.docker"
-mkdir -p "$DOCKER_CONF"
-docker --config="$DOCKER_CONF" login -u="$QUAY_USER" -p="$QUAY_TOKEN" quay.io
-docker --config="$DOCKER_CONF" login -u="$RH_REGISTRY_USER" -p="$RH_REGISTRY_TOKEN" registry.redhat.io
-docker --config="$DOCKER_CONF" build -t "${IMAGE}:${IMAGE_TAG}" .
-docker --config="$DOCKER_CONF" push "${IMAGE}:${IMAGE_TAG}"
+# Install bonfire repo/initialize
+CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
+curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
 
-#
-# Install Bonfire and dev virtualenv
-#
+source $CICD_ROOT/build.sh
 
-if [ ! -d bonfire ]; then
-    git clone https://github.com/RedHatInsights/bonfire.git
-fi
+source $APP_ROOT/unit_test.sh
 
-if [ ! -d venv ]; then
-    python3 -m venv venv
-fi
+#source $CICD_ROOT/deploy_ephemeral_env.sh
```

Comment:
Can these commented lines be removed?

Reply:
We could. I'm using it as a place-holder related to running smoke tests to show at which point in the pr_check.sh it should be placed, we'll be un-commenting it soon

Reply:
Removed them